modules/ROOT/pages/about-my-applications.adoc

```diff
@@ -9,17 +9,37 @@ applications that hold a contract between an API and a user.
 
 image::ex2-my-applications.png[]
 
-A contract holds information about the contract with a client ID and client secret for the API.
-Each application lists a description, its URL, redirect URIs, grant types, and a usage dashboard.
+A contract holds information including a client ID and optional client secret
+for the API. Each application lists a description, its URL, redirect URIs,
+grant types, and a usage dashboard.
```

Comment:
Does the contract hold information FOR the API or ABOUT the API?
Would it be just as accurate to say:
"A contract includes information about an API, including its client ID; its client secret, if any; its description; its URIs; its grant types, and its usage dashboard"?
Or are these two sentences about two different things: the contract that includes ID and secret, and the API that includes the description etc.?
Clarify.

Reply:
A contract is the link between a client application and an API. An API can have only one contract with a specific client application. A client application can have several contracts with different APIs. The only thing a contract may hold is SLA tier info and terms and conditions. A client application is identified by a client ID and client secret (both mandatory and generated by the platform). A client application holds as well, description , grant types supported, etc. Usage (aka analytics) is stored in a different place.